commands/publish_dafsa.py

```diff
@@ -0,0 +1,64 @@
+import requests
+from requests.exceptions import HTTPError
+import wget
+import tempfile
+import subprocess
+
+COMMIT_HASH_URL = "https://api.github.com/repos/publicsuffix/list/commits/master"
+LIST_URL = (
+    "https://raw.githubusercontent.com/publicsuffix/list/master/public_suffix_list.dat"
+)
+MAKE_DAFSA_PY = "https://raw.githubusercontent.com/arpit73/temp_dafsa_testing_repo/master/publishing/make_dafsa.py"
```

Comment:
I'm using a temporary repository for storing the modified versions of make_dafsa.py and prepare_tlds.py, the links can be switched to mozilla-central when the patches land.